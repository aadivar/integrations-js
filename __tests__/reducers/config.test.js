import {
  setConfig,
} from '../../src/actions/config';

import configReducer from '../../src/reducers/config';

test('configReducer default', () => {
  expect(configReducer(undefined, {})).toMatchSnapshot();
});

test('configReducer data', () => {
  expect(configReducer(undefined, setConfig({ siteId: 'pk_efh893f' }))).toMatchSnapshot();
});
