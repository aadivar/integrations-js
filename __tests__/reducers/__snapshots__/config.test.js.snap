// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configReducer data 1`] = `
Object {
  "siteId": "pk_efh893f",
}
`;

exports[`configReducer default 1`] = `null`;
